go/rpc/encrypted_client.go

```diff
@@ -147,45 +147,53 @@ func (c *EncRPCClient) Subscribe(ctx context.Context, namespace string, ch inter
 		return nil, fmt.Errorf("expected a channel of type `chan types.Log`, got %T", ch)
 	}
 	clientChannel := make(chan common.IDAndEncLog)
-	subscription, err := c.obscuroClient.Subscribe(ctx, namespace, clientChannel, subscriptionType, encryptedParams)
+	subscription, err := c.obscuroClient.Subscribe(ctx, nil, namespace, clientChannel, subscriptionType, encryptedParams)
 	if err != nil {
 		return nil, err
 	}
 
-	go func() {
```

Comment:
The goroutine has been pulled out into a new `forwardLogs` method. I wouldn't have done the refactoring in this PR, but the linter was complaining about cyclomatic complexity, and you know I hate cyclomatic complexity.

Reply:
It's my third least-favourite kind of complexity